pages/resources/quizzes/firstLanguage/FirstProgrammingLanguageQuiz.tsx

```diff
@@ -1,3 +1,4 @@
+import { useMutation } from "@apollo/client/react/hooks/useMutation";
```

Comment:
let's actually just name this file index.tsx and the folder firstProgrammingLanguage

you'll be able to navigate to the quiz like this: skillify.ca/resources/quizzes/firstProgrammingLanguage

If you're curious about how routing works in next JS: https://nextjs.org/docs/routing/dynamic-routes